public/css/bootstrapV4.css

```diff
@@ -0,0 +1,1490 @@
+/*!
+ * Bootstrap v4.0.0-alpha (http://getbootstrap.com)
+ * Copyright 2011-2015 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+
+@charset "UTF-8";
+/*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+img {
+  vertical-align: middle;
+}
+
+.img-responsive, .figure > img, .carousel-inner > .carousel-item > img,
+.carousel-inner > .carousel-item > a > img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+.img-rounded {
+  border-radius: .3rem;
+}
+
+.img-thumbnail {
+  display: inline-block;
+  max-width: 100%;
+  height: auto;
+  padding: .25rem;
+  line-height: 1.5;
+  background-color: #fff;
+  border: 1px solid #ddd;
+  border-radius: .25rem;
+  -webkit-transition: all .2s ease-in-out;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The property -moz-transition is compatible with -webkit-transition and -o-transition and should be included as well.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2386098985/issues/source?bid=3148782&fileBranchId=3227421#l33)